GOVERNANCE.md

```diff
@@ -0,0 +1,133 @@
+
+WG seats are not time-limited. There is no fixed size of the WG.
+
+There is no specific set of requirements or qualifications
+for WG membership beyond these rules.
+
+The WG may add additional members to the WG by unanimous consensus.
+
+A WG member may be removed from the WG by voluntary resignation,
+or by unanimous consensus of all other WG members. If a member is
+removed from the WG then they are also removed from all WG teams
+(including the Releasers team).
+
+Changes to WG membership should be posted in the agenda, and may be
+suggested as any other agenda item (see "WG Meetings" below).
+
+If an addition or removal is proposed during a meeting, and the full WG
+is not in attendance to participate, then the addition or removal is
+added to the agenda for the subsequent meeting. This is to ensure
+that all members are given the opportunity to participate in all
+membership decisions. If a WG member is unable to attend a meeting
+where a planned membership decision is being made,
+then their consent is assumed.
+
+No more than 1/3 of the voting WG members may be affiliated with the same
+employer. If removal or resignation of a WG member, or a change of
+employment by a WG member, creates a situation where more than 1/3
+of the WG membership shares an employer, then the situation must be
+immediately remedied by the resignation or removal of one or more
+WG members affiliated with the over-represented employer(s).
+
+## WG Meetings
+
+The WG meets every 3 weeks on a Google Hangout On Air. One of the
+WG members will volunteer to act as the moderator for each meeting
+subject to agreement from the rest of the members. Each meeting should be
+published to YouTube.
+
+Members of the `releasers` sub-team are not required to attend
+the WG meetings and maintain their WG membership simply by
+participating in the release process at least once a year.
+
+Members of the `CitGM` sub-team are not required to attend
+the WG meetings and maintain their WG membership simply by
+participating in the CitGM repository at least once a year.
+
+Items are added to the WG agenda that are considered contentious or are
+modifications of governance, contribution policy,
+WG membership, or release process.
+
+The intention of the agenda is not to approve or review all patches;
+that should happen continuously on GitHub and be handled
+by the larger group of Collaborators.
+
+Any community member or contributor can ask that something be
+added to the next meeting's agenda by logging a GitHub Issue.
+Any Collaborator, WG member or the moderator can add the item
+to the agenda by adding the WG-agenda tag to the issue.
+
+Prior to each WG meeting the moderator will share the Agenda with
+members of the WG. WG members can add any items they like to the
+agenda at the beginning of each meeting. The moderator and the WG
+cannot veto or remove items.
+
+The WG may invite persons or representatives from certain
+projects to participate in a non-voting capacity.
```

Comment:
I think people can always ask.   In terms of not being too specific I think I'd leave that out from here.